account_invoice_date/wizard/account_voucher_proforma_date.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2022 Thomas Rehn (initOS GmbH)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
```

Comment:
Unused import and variable, and `api` not used

Reply:
Removed unnecessary imports